docs/core-protocol/smart-contracts-api/PublicLock.md

```diff
@@ -1242,37 +1224,23 @@ Update transfer and cancel rights for a given key
 | _tokenId | uint256 | The id of the key to assign rights for |
 | _keyManager | address | The address to assign the rights to for the given key |
 
-### setMaxKeysPerAddress
+### setLockMetadata
 
 ```solidity
-function setMaxKeysPerAddress(uint256 _maxKeysPerAddress) external nonpayable
+function setLockMetadata(string _lockName, string _lockSymbol, string _baseTokenURI) external nonpayable
 ```
 
-Set the maximum number of keys a specific address can use
+Allows the Lock owner to assign 
 
 
 
 #### Parameters
 
 | Name | Type | Description |
 |---|---|---|
-| _maxKeysPerAddress | uint256 | the maximum amount of key a user can own |
-
-### setMaxNumberOfKeys
-
-```solidity
-function setMaxNumberOfKeys(uint256 _maxNumberOfKeys) external nonpayable
```

Comment:
Are we really removing this?

Reply:
Yup. All happening thru `setLockMetadata`